README.md

```diff
@@ -8,12 +8,17 @@ This can be run locally using the Procfile or via Docker through the Dockerfile.
 
 To build the Dockerfile, run:
 ```
-docker build -f Dockerfile.dev -t exercism-tooking-orchestrator .
+docker build -f Dockerfile.dev -t exercism-tooling-orchestrator .
 ```
 
 To execute the Dockerfile, run the following with your AWS keys:
 ```
-docker run -p 3021:3021 -v /PATH/TO/PWD:/usr/src/app exercism-tooking-orchestrator:latest
+docker run -p 3021:3021 -v /PATH/TO/PWD:/usr/src/app exercism-tooling-orchestrator:latest
+```
+
+For example:
+```
+docker run -p 3021:3021 -v /Users/iHiD/Code/exercism/tooling-orchestrator:/usr/src/app exercism-tooling-orchestrator:latest
 ```
```

Comment:
The AWS part is outdated now, right? And the user also needs to volume map the `/usr/src/settings/settings.yml` file, right?